apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CollectionAssertionSamples.kt

```diff
@@ -0,0 +1,71 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.hasSize
+import ch.tutteli.atrium.api.fluent.en_GB.isEmpty
+import ch.tutteli.atrium.api.fluent.en_GB.isGreaterThan
+import ch.tutteli.atrium.api.fluent.en_GB.isLessThan
+import ch.tutteli.atrium.api.fluent.en_GB.isNotEmpty
+import ch.tutteli.atrium.api.fluent.en_GB.size
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class CollectionAssertionSamples {
+
+    @Test
+    fun isEmpty() {
+        expect(listOf<Int>()).isEmpty()
+
+        fails {
+            expect(listOf(1, 2, 3)).isEmpty()
+        }
+    }
+
+    @Test
+    fun isNotEmpty() {
+        expect(listOf(1, 2, 3)).isNotEmpty()
+
+        fails {
+            expect(listOf<Int>()).isNotEmpty()
+        }
+    }
+
+    @Test
+    fun hasSize() {
+        expect(listOf(1, 2, 3)).hasSize(3)
+
+        fails {
+            expect(listOf(1, 2, 3)).hasSize(1)
+        }
+    }
+
+    @Test
+    fun sizeFeature() {
+        expect(listOf(1, 2, 3)).size
```

Comment:
See samples for pairAssertions. Please document the subject change and the difference between size and size{} etc

Reply:
Ok ! I push some changes